## 23% of end-end tests have passed
```
Total executed tests: 56
Failed tests: 43
Expected to fail tests: 0
Passed tests: 13
```
```
IDs of failed tests: 702845,702840,702894,703086,702855,702947,702730,702958,702786,702745,702731,702844,702851,703202,741054,702841,703496,702807,702839,702733,702808,702809,702838,702813,704615,702948,702859,703629,702957,702783,702843,702782,702850,740220,702784,703391,740221,702732,703297,703495,702777,703194,740222 
```
<h3>Failed tests (43)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=tests/user-to-update-profile&PR_ID=21982&APK_NAME=21982.apk&TR_CASE_IDS=702845,702840,702894,703086,702855,702947,702730,702958,702786,702745,702731,702844,702851,703202,741054,702841,703496,702807,702839,702733,702808,702809,702838,702813,704615,702948,702859,703629,702957,702783,702843,702782,702850,740220,702784,703391,740221,702732,703297,703495,702777,703194,740222">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772316">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:866: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772305">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `contact-request-message` </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:866: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES9wj12QCRCWSCJjMK">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESdnJvmoJs3QPyJWzN">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772308">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:866: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772325">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:866: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772338">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:866: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Can't navigate to a community channel before joining the community, https://github.com/status-im/status-mobile/issues/21776]]</b><br/><br/></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772336">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:569: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772303">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `contact-request-message` </div></blockquote></p>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:569: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEScRx2CTa3Q2l0KwHk">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESThepb7rAmilNew9k">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772304">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:569: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772324">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `contact-request-message` </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:250: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESpus59BHQzXI5ar66">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESrj7WxethOJjhY5Nk">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772312">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772323">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772328">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772315">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Youtube links preview is not loaded on LambdaTest emulators, needs investigation]]</b><br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772313">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772311">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772310">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `contact-request-message` </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES7uUDyF9UYjqXxa2z">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESi3tAeW9Is2AO2CtE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772340">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772321">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772314">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772330">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:324: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772331">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:60: in prepare_devices
    self.homes[0].add_contact(self.public_keys[i])
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772306">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `contact-request-message` </div></blockquote></p>

```
Test setup failed: critical/chats/test_group_chat.py:60: in prepare_devices
    self.homes[0].add_contact(self.public_keys[i])
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEScctZNBWvDZSTBu9A">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESOpBPZSYGCJ1taJYv">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESGCUkRZ6Xb4ac2gJw">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772307">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:60: in prepare_devices
    self.homes[0].add_contact(self.public_keys[i])
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772296">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:60: in prepare_devices
    self.homes[0].add_contact(self.public_keys[i])
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772332">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:60: in prepare_devices
    self.homes[0].add_contact(self.public_keys[i])
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772335">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:60: in prepare_devices
    self.homes[0].add_contact(self.public_keys[i])
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772327">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:393: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772326">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `contact-request-message` </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:393: in prepare_devices
    self.home_1.add_contact(self.public_key_2)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESjjv5vngug55PLDWv">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES3QD1yk1br72u2TGh">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772320">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:36: in prepare_devices
    self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772294">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `contact-request-message` </div></blockquote></p>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:36: in prepare_devices
    self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESOBHhxgAxgJHCWs2U">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESX5uK5033NhDCEdGD">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772299">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:36: in prepare_devices
    self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772295">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:36: in prepare_devices
    self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772297">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:36: in prepare_devices
    self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772309">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:36: in prepare_devices
    self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772302">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:36: in prepare_devices
    self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772334">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:36: in prepare_devices
    self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772319">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `new-chat-button` </div><div>Device 2: Find `Button` by `accessibility id`: `new-chat-button` </div></blockquote></p>

```
activity_center/test_activity_center.py:105: in test_activity_center_contact_request_accept_swipe_mark_all_as_read
    self.home_2.new_chat_button.click_until_presence_of_element(self.home_2.add_a_contact_chat_bottom_sheet_button)
../views/base_element.py:109: in click_until_presence_of_element
    raise NoSuchElementException("%s element not found" % desired_element.name)
 Button element not found; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaR6jJdoJ7GbDZcjx">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES3w361yRQsc4CiSoe">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772318">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `new-chat-button` </div><div>Device 2: Find `Button` by `accessibility id`: `new-chat-button` </div></blockquote></p>

```
activity_center/test_activity_center.py:33: in test_activity_center_contact_request_decline
    self.home_2.new_chat_button.click_until_presence_of_element(self.home_2.add_a_contact_chat_bottom_sheet_button)
../views/base_element.py:109: in click_until_presence_of_element
    raise NoSuchElementException("%s element not found" % desired_element.name)
 Button element not found; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaR6jJdoJ7GbDZcjx">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES3w361yRQsc4CiSoe">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772301">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: LogInButton </div><div>## Signed in successfully!</div></blockquote></p>

```
activity_center/test_activity_center.py:167: in test_add_contact_field_validation
    new_username_1, _ = self.loop.run_until_complete(run_in_parallel(((_device_1_creates_user, {}),
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
activity_center/test_activity_center.py:158: in _device_1_creates_user
    self.home_1.reopen_app(sign_in=False)
../views/base_view.py:341: in reopen_app
    self.driver.terminate_app(app_package)
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/appium/webdriver/extensions/applications.py:196: in terminate_app
    return self.assert_extension_exists(ext_name).execute_script(
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@2@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'com.google.android.apps.nexuslauncher' is still running after 500ms timeout
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'com.google.android.apps.nexuslauncher' is still running after 500ms timeout
E       at getResponseForW3CError (/root/lambda/shared-tools/volume/appium/appium-2.1.3/node_modules/@appium/base-driver/lib/protocol/errors.js:1097:9)
E       at asyncHandler (/root/lambda/shared-tools/volume/appium/appium-2.1.3/node_modules/@appium/base-driver/lib/protocol/protocol.js:491:57)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaR6jJdoJ7GbDZcjx">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES3w361yRQsc4CiSoe">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772349">test_fallback_add_key_pair</a>, id: 741054 </b></td></tr><tr><td>

```
Test setup failed: critical/test_fallback.py:38: in prepare_devices
    self.home_1.add_contact(self.public_key_3)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772345">test_fallback_sync_with_error</a>, id: 740220 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `EditBox` by `accessibility id`: `contact-request-message` </div></blockquote></p>

```
Test setup failed: critical/test_fallback.py:38: in prepare_devices
    self.home_1.add_contact(self.public_key_3)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772346">test_fallback_with_correct_seed_phrase</a>, id: 740221 </b></td></tr><tr><td>

```
Test setup failed: critical/test_fallback.py:38: in prepare_devices
    self.home_1.add_contact(self.public_key_3)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772347">test_fallback_validate_seed_phrase</a>, id: 740222 </b></td></tr><tr><td>

```
Test setup failed: critical/test_fallback.py:38: in prepare_devices
    self.home_1.add_contact(self.public_key_3)
../views/home_view.py:424: in add_contact
    chat.contact_request_message_input.send_keys("hi")
../views/base_element.py:365: in send_keys
    self.find_element().send_keys(value)
../views/base_element.py:78: in find_element
    raise NoSuchElementException(
 Device 1: EditBox by accessibility id: `contact-request-message` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (13)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772343">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772348">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772329">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuVIoPHKRanbVCmvq">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772298">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuVIoPHKRanbVCmvq">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772317">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuVIoPHKRanbVCmvq">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772322">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuVIoPHKRanbVCmvq">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772333">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuVIoPHKRanbVCmvq">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772337">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuVIoPHKRanbVCmvq">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772342">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772341">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772339">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESdu9YIak6broYn0Cu">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772300">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESdu9YIak6broYn0Cu">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3772344">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESdu9YIak6broYn0Cu">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/test/appium/tests/critical/chats/test_1_1_public_chats.py
+++ b/test/appium/tests/critical/chats/test_1_1_public_chats.py
@@ -32,7 +32,8 @@ class TestOneToOneChatMultipleSharedDevicesNewUi(MultipleSharedDeviceTestCase):
         for home in (self.home_1, self.home_2):
             home.navigate_back_to_home_view()
             home.chats_tab.click()
-        self.home_1.add_contact(self.public_key_2)
+        self.username_1 = "user_1"
+        self.home_1.add_contact(public_key=self.public_key_2, username=self.username_1)
 
         self.home_2.just_fyi("Accepting contact request from activity centre")
         self.home_2.handle_contact_request(self.username_1)
@@ -243,7 +244,7 @@ class TestOneToOneChatMultipleSharedDevicesNewUi(MultipleSharedDeviceTestCase):
             unpin_element.click_until_absense_of_element(desired_element=unpin_element)
             self.chat_1.pin_message(self.message_4, 'pin-to-chat')
             for chat in self.chat_1, self.chat_2:
-                if chat.chat_element_by_text(self.message_4).pinned_by_label.is_element_displayed(30):
+                if not chat.chat_element_by_text(self.message_4).pinned_by_label.is_element_displayed(30):
                     self.errors.append(chat, "Message 4 is not pinned in chat after unpinning previous one")
 
         self.home_1.just_fyi("Check pinned messages are visible in Pinned panel for both users")

--- a/test/appium/views/home_view.py
+++ b/test/appium/views/home_view.py
@@ -247,7 +247,9 @@ class HomeView(BaseView):
     def __init__(self, driver):
         super().__init__(driver)
 
-        self.plus_button = Button(self.driver, accessibility_id="new-chat-button")
+        self.skip_editing_profile_button = Button(self.driver, xpath="//*[@content-desc='button-two']//*[@text='Skip']")
+        self.edit_profile_button = Button(self.driver, xpath="//*[@content-desc='button-one']//*[@text='Edit Profile']")
+        self.submit_create_profile_button = Button(self.driver, accessibility_id="submit-create-profile-button")
         self.plus_community_button = Button(self.driver, accessibility_id="new-communities-button")
         self.chat_name_text = Text(self.driver, accessibility_id="chat-name-text")
         self.start_new_chat_button = ChatButton(self.driver, accessibility_id="start-1-1-chat-button")
@@ -400,9 +402,15 @@ class HomeView(BaseView):
             self.close_activity_centre.wait_for_rendering_ended_and_click()
             self.chats_tab.wait_for_visibility_of_element()
 
-    def add_contact(self, public_key, nickname=''):
+    def add_contact(self, public_key, username='', nickname=''):
         self.driver.info("Adding user to Contacts via chats > add new contact")
-        self.new_chat_button.click_until_presence_of_element(self.add_a_contact_chat_bottom_sheet_button)
+        self.new_chat_button.click()
+        if username:
+            self.edit_profile_button.click()
+            self.get_sign_in_view().profile_title_input.send_keys(username)
+            self.submit_create_profile_button.click()
+        else:
+            self.skip_editing_profile_button.click_if_shown()
         self.add_a_contact_chat_bottom_sheet_button.click()
 
         chat = self.get_chat_view()

